src/components/App.tsx

```diff
@@ -770,7 +772,9 @@ class App extends React.Component<AppProps, AppState> {
         },
       });
     }
-
+    this.nearestScrollableContainer = getNearestScrollableContainer(
+      this.excalidrawContainerRef.current!,
+    );
```

Comment:
I'm getting around 10-50ms on this because it seems it triggers a document reflow. If we defer it (setTimeout), moving it into the `addEventListeners()`, the reflow issue seems to be gone and I get sub-millisecond perf.